Dockerfile-alpine

```diff
@@ -0,0 +1,57 @@
+FROM php:7.0-alpine
+
+MAINTAINER Tobias Munk tobias@diemeisterei.de
```

Comment:
`MAINTAINER` is deprecated use `LABEL` instead

https://docs.docker.com/engine/userguide/labels-custom-metadata/
https://docs.docker.com/engine/reference/builder/#label